### PR TITLE
Ethics society newsletter 2

### PR DESCRIPTION
**Pull request by yjernite**
Second seasonal newsletter of the Ethics and Society team, target release Thursday 12.15

### CODE REVIEW

File: ethics-soc-2.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
+---
+title: "Ethics and Society Newsletter #2" 
+thumbnail: /blog/assets/103_ethics-soc-1/thumbnail.png
+---
+
```
Comment (osanseviero) [inline ethics-soc-2.md:5]: Except for the thumbnail, we now try to put all assets in the hf.co site. That way we avoid having this repository being too large to use for others https://huggingface.co/datasets/huggingface/documentation-images/tree/main/blog

**Review by osanseviero — APPROVED** [review]

File: ethics-soc-2.md (hunk 1 above)
Comment (yjernite) [reply ethics-soc-2.md:5]: Thanks, missed that!

Is it OK to delete them in this branch or should I make a new PR without the history?

Comment (osanseviero) [reply ethics-soc-2.md:5]: Deleting should be fine! When we merge, we can squash the commits so at the end we won't have the images in the history

**Comment by yjernite** [comment]
Added the thumbnail and `_blog.yml` entry and moved the assets to the Hub, ready to go on my end!

**Pull request merged**